app/proc/processor.go

```diff
@@ -142,3 +155,18 @@ func (p *Processor) setDefaults() {
 		p.Conf.System.UpdateInterval = time.Minute * 5
 	}
 }
+
+func (filter *Filter) skip(item feed.Item) (bool, error) {
+	if filter.Title != "" {
+		matched, err := regexp.MatchString(filter.Title, item.Title)
+		if err != nil {
+			log.Printf("[WARN] wrong title's regular expression: %s", filter.Title)
```

Comment:
Error handling shouldn't be mixed like this. The func should handle it or by returning an error (with necessary context) or by "handling" it (log.Print in your case), but not both.